src/components/media-views.js

```diff
@@ -336,12 +345,29 @@ AFRAME.registerComponent("media-video", {
         // TODO FF error here if binding mediastream: The captured HTMLMediaElement is playing a MediaStream. Applying volume or mute status is not currently supported -- not an issue since we have no audio atm in shared video.
         texture.audioSource = this.el.sceneEl.audioListener.context.createMediaElementSource(texture.image);
 
-        const sound = new THREE.PositionalAudio(this.el.sceneEl.audioListener);
-        sound.setNodeSource(texture.audioSource);
-        this.el.setObject3D("sound", sound);
+        let audio;
+
+        if (this.data.audioType === "pannernode") {
```

Comment:
Hmm, does the fact that its a PannerNode actually matter here? I feel like it should just be "positional" or "spatialized"

Reply:
Aren't these properties just one model of spatial audio though? Like Google's resonance API has some additional/different properties but could fall back to the panner node: https://developers.google.com/resonance-audio/reference/web/Source#~SourceOptions

Reply:
I was thinking of them as pretty generic, but I guess its true that not all spatialization is going to have properties that look like this.